### PR TITLE
chore(website): enhance plugin search ui

### PR DESCRIPTION
**Pull request by mbtools**
- Change navigation and page title to "Search for Plugins and Tools"
- Add Verdaccio logo to core plugins
- Show "type of plugin" on icon hover
- Clarify that the number is "Monthly downloads"
- Clarify that version is the "Latest version"
- Clarify that "Visit" opens npmjs.com

Ref #3539

Some screenshots:

image

image

image

image

image

image

Also 

- Remove two duplicate entries in `addons.json` which caused weird search results:

  image

- Replace any HTML tags and markdown links in addon descriptions in `addon-update.ts` to fix these:
  
  image
  
  image

PS: This was a nice opportunity to learn more about react. And GH copilot... that's just killer!

### CODE REVIEW

**Comment by mbtools** [comment]
Open questions:

1. How do you localize attributes of react elements?
2. npm-offline-packager is listed as an add-on but does not exist (anymore?). Should it be removed?

File: .changeset/popular-nails-behave.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
Comment (juanpicado) [inline .changeset/popular-nails-behave.md:1]: no need changes for website, has no version

**Comment by juanpicado** [comment]
> 2\. n

Number 2, yes please.

> How do you localize attributes of react elements with docusaurus?

do you mean languages? if yes, so far only EN ... I reduced the amount of languages render by docusaurus because most of them had 15% coverage and I move the bar to 80% otherwise is merely English everywhere anyway, so this page could be only on English no need translations.

**Comment by mbtools** [comment]
It's just texts you see on hover so English only is ok 👍

**Review by juanpicado — APPROVED** [review]
Great work !! thanks !

**Pull request merged**